transcribed/page1200.txt

```diff
@@ -16,45 +16,45 @@ GTYOCN   079 006   ITRN1    026 022   JDF8     112 065   MEMTOP   118 041   NOGO
 GTYOR1   074 032   ITRN4    026 040   JDS4     115 028   MGCC     078 037   NTTYS    060 030   SAC17P   118 018
-GTYOST   079 024   ITRN5    026 054   JNAME    118 045   MGCDP    078 036   NULB1    052 019   SBRK1    009 038
+GTYOST   079 024   ITRN5    026 054   JNAME    118 045   MGCDP    078 036   NULBI    052 019   SBRK1    009 038
 GTYOT1   079 007   ITYI     019*020   L        118*063   MGCR2    063 049   NULBOD   052 026   SBRK2    009 046
 GTYOT2   079 017   IUTC1    095 022   LCDIY    021 047   MGCR3    063 020   NULI     052 014   SBRK69   008 037
 GTYOT3   079 018   IUTC2    095 018   LCPDL    117 022   MGCR4    063 037   NULJOB   017 038   SCLKI    002 024
 GTYOT4   079 019   IUTC3    095 007   LINEPO   059 023   MGG1     064 017   NULO     052 003   SCOR     116 003
 GTYOT5   079 020   IUTCO1   095 003   LISTEN   019*021   MGTCR1   062 034   NULOD    052 017   SCOREX   116 017
-GTYOT7   079 021   IUTCON   095 002   LLTPSN   047 027   MGTCR2   063 053   NULOO    052 017   SCRFG1   117 025
-GTYOW1   079 021   J        001 014   LOGIN    019*025   MGTCR3   063 010   NUNITS   103 005   SCRFLG   117 024
-GTYT1    072 032   JDB3     111 002   LPC      059 008   MGTF     063 002   NUTCA    083 052   SDCHN    002 019
+GTYOT7   079 021   IUTCON   095 002   LLTPSN   047 027   MGTCR2   063 053   NULO0    052 009   SCRFG1   117 025
+GTYOW1   061 035   J        001 014   LOGIN    019*025   MGTCR3   063 010   NUNITS   103 005   SCRFLG   117 024
+GTYT1    072 032   JDB3     111 002   LPC      059 008   MGTF     064 002   NUTCA    083 052   SDCHN    002 019
 HGCBT    065*002   JDB3A    114 022   LPC1     058 029   MGTF1    064 005   NUTIC    083 004   SEAR2    009 013
-HGCBTL   065 006   JDB3B    115 003   LPC2     059 018   MGTF2    063 050   NUTOC    082 005   SEARC    009 021
+HGCBTL   065 006   JDB3B    115 003   LPC2     059 018   MGTF2    063 050   NUTOC    083 005   SEARC    009 021
 HTYI     065 034   JDB4     113*009   LPEN     019*036   MGTYCR   063 002   ONEFLS   021 002   SEARCH   008 003
-HTYO     065 039   JDB4A    113 013   PLENT    066 007   MGTYO    062 014   OPEN     019 004   SEARE    009 033
+HTYO     065 039   JDB4A    113 013   LPENT    066 007   MGTYO    062 014   OPEN     019 004   SEARE    009 003
 I        001 012   JDB4B    113 011   LPENT1   059 040   MGTYO1   063 006   OPEN1    024 058   SEARL    008 012
 IBRTN    095 027   JDB5     113 002   LPTA     059 063   MGTYOE   062 042   OPEN2    024 047   SEARLP   008 018
-IGCK     074*012   JDB6     099 035   LPTBE    059 059   MGTYOR   062 038   OPER     019 005   SEARTN   009 005   
-ILLOP    022 012   JDB6A    099 003   LPTBO    059 058   MGTYT1   064 023   OPRDSP   027 036   SEARW    009 023
+IGCK     074*012   JDB6     099 035   LPTBE    059 059   MGTYOR   062 038   OPER     019 005   SEARTN   009 005
+ILLOP    022 012   JDB6A    099 003   LPTB0    059 058   MGTYT1   064 023   OPRDSP   027 036   SEARW    009 023
```

Comment:
I think this is an O rather than 0, because it's probably "light pen binary open".

Reply:
Aha! Sounds probable.